### DISCUSSION TITLE
[draw] draw on video in real time

### DISCUSSION THREAD

**Discussion by williamfjm** [raysan5/raylib#2712] [Q&A]
hello all again .i have used raylib for a while for some small projects .
But due to the scarcity of raspberrys, I had to migrate my software to some arm tv boxes that do not have omxplayer or dispmax.
so i had to switch to mplayer as player .but i need to draw on the video in real time .

so I wonder if it is possible to leave a totally transparent zone in raylib to let the video run without the "ontop" option.

I already use pl_mpeg and the performance is very poor.

Or how else can I render an image on top of this playback?

I apologize if this is not the ideal place to ask the question.
Thank you .

**Comment by raysan5** [comment]
Afaik, `pl_mpeg` is one of the best options out there for software video frames decompresion. Did you measure the cost of frame-decompression? Drawing an overlay over the frame with raylib should have a very low cost so probably the bottleneck is on the video frame decompresion.
